mypy_json_report/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Memrise
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .cli import main
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
**Suggestion:** This should probably be moved to `__main__.py`? That's the file which is executed if you are to run `python -m mypy_json_report`. I don't think it really serves any purpose here.